lib/ansible/modules/cloud/digital_ocean/do_sshkeys.py

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/python
```

Comment:
The rename of this module isn't great - it breaks the convention of prefixing with `digital_ocean_`, would it be possible to name this `digital_ocean_ssh_keys` or `digital_ocean_sshkeys` so it matches the rest?